pom.xml

```diff
@@ -194,6 +195,11 @@
             <artifactId>gson</artifactId>
             <version>${gsonVersion}</version>
         </dependency>
+        <dependency>
```

Comment:
You should  exclude the dependency from the library that takes this one as transitive so we are should that this version is picked.